static/js/components/ProfileImage.js

```diff
@@ -7,18 +7,41 @@ import type { Profile } from '../flow/profileTypes';
 
 export default class ProfileImage extends React.Component {
   props: {
-    profile: Profile
+    profile: Profile,
+    editable: boolean
   };
 
+  static defaultProps = {
+    editable: false
+  };
+
+  cameraSvgImage: Function = (): React$Element<*> => {
+    return (
+      <svg fill="#FFFFFF" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+        <circle cx="12" cy="12" r="3.2"/>
+        <path d={`M9 2L7.17 4H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 
+          2-2V6c0-1.1-.9-2-2-2h-3.17L15 2H9zm3 15c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5z`}/>
+        <path d="M0 0h24v24H0z" fill="none"/>
+      </svg>
+    );
+  }
```

Comment:
we set `i.material-icons` to have `color: $mm-brand-blue`. you should be able to override that for a particular icon by using a more specific selector (like `i.material-icons.user-info-edit` or something). In that case I think your `cameraIcon` function would directly return the `<Icon>` component.